compiler/optimizer/StructuralAnalysis.hpp

```diff
@@ -57,10 +57,10 @@ class TR_RegionAnalysis
    private:
 
    class StructInfo;
-   typedef CS2::TableOf<StructInfo, TR::Allocator> InfoTable;
-   typedef TR::BitVector StructureBitVector;
-   typedef TR::BitVector WorkBitVector;
-   typedef TR::deque<TR_StructureSubGraphNode*> SubGraphNodes;
+   typedef StructInfo** InfoTable;
+   typedef TR_BitVector StructureBitVector;
+   typedef TR_BitVector WorkBitVector;
+   typedef TR::deque<TR_StructureSubGraphNode*, TR::Region&> SubGraphNodes;
 
```

Comment:
Are the `#include "cs2/arrayof.h"` and `#include "cs2/tableof.h"` still required in this file?  Please remove if not.